server/src/main/resources/db/R__Load_testing_data.sql

```diff
@@ -0,0 +1,143 @@
+delete from action_items;
```

Comment:
This script needs to be set up to not run in all environments. Should only run in local and dev. See https://github.com/oci-labs/check-ins/pull/297/files#diff-09185e226a8b5c7c467a55d4722011ed for examples of how to do this.

Reply:
done